environments/staging/service-core-compute-aat-internal.yml

```diff
@@ -21,4 +21,10 @@ vnet_links:
   - link_name: "bastion-prod-vnet"
     vnet_id: "/subscriptions/2b1afc19-5ca9-4796-a56f-574a58670244/resourceGroups/bastion-prod-rg/providers/Microsoft.Network/virtualNetworks/bastion-prod-vnet"
 A: []
-cname: []
+cname:
+  - name: neuvector00
+    record: 10.10.143.250
```

Comment:
this should be an A record and not a CNAME record

Reply:
ammended